experiment/generate_tests.py

```diff
@@ -0,0 +1,233 @@
+#!/usr/bin/env python
+
+# Copyright 2017 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Create e2e test definitions.
+
+Usage example:
+
+  In $GOPATH/src/k8s.io/test-infra,
+
+  $ ./experiment/generate_tests.py \
+      --yaml-config-path=experiment/test_config.yaml \
+      --json-config-path=jobs/config.json
+"""
+
+import argparse
+import hashlib
+import json
+import os
+import yaml
+
+# pylint: disable=too-many-branches,too-many-statements,too-many-locals
```

Comment:
We should not have module level disables. Either stick it into the pylint file or else silence it at a lower scope